mod-source-record-manager-server/src/main/java/org/folio/verticle/DataImportJournalBatchConsumerVerticle.java

```diff
@@ -421,11 +421,13 @@ private Completable commitOffset(Map<TopicPartition, OffsetAndMetadata> offsets)
       )
       .onErrorResumeNext(error -> {
         if (error instanceof RebalanceInProgressException) {
-          LOGGER.warn("Rebalance in progress. Retrying offset commit...");
-          return kafkaConsumer.rxPoll(Duration.ofMillis(100))
-            .flatMapCompletable(records -> commitOffset(offsets));
```

Comment:
but here we were just polling tp finish the rebalance and committing the same offsets? 

`2025-01-18T15:28:15.450588353Z org.apache.kafka.common.errors.RebalanceInProgressException: Offset commit cannot be completed since the consumer is undergoing a rebalance for auto partition assignment. You can try completing the rebalance by calling poll() and then retry the operation.`

Reply:
Remove commit of offsets as discussed if it is generating an error.